### ISSUE TITLE
Camera controls

### ISSUE THREAD

**Issue by Serenitymae** [PrehistoricKingdom/feedback#501]
This game really needs to be able to rotate the camera. It's just stuck on one angle with no option to rotate the camera. We can rotate just fine, but not the camera. Can't play a simulation game like that. It's pretty annoying.

**Comment by CuPanda63** [comment]
I will also add being able to map mouse controls differently would be really helpful.

**Comment by Kassirex** [comment]
yeah i cant rotate my camera either right now, its just stuck looking in one direction, it makes the game un playable and theres no way to fix it or rebind it with how things are
hopefully the devs can get to this when they finish with some of the other bugs

**Comment by aviboys** [comment]
je suis également dans l'incapacité de tournée la caméra. ces très agacent. jeux dit terminé qui ne les pas....

**Comment by CosmicGlowsticks** [comment]
Yeah, I'm stuck on the tutorial, it sucks. I don't have a mouse at all, and despite having a setting on to where 3 fingers = middle mouse, it literally doesn't work for anything. Unplayable for me, too. I need it to be rebindable to keys, like < and >

**Comment by Potosz** [comment]
I can't rotate the camera. And so it's impossible to play.

**Comment by Kassirex** [comment]
this needs a fix as soon as it can be done
